backend/src/main/java/com/backend/Controller/FavoriteBookController.java

```diff
@@ -0,0 +1,38 @@
+package com.backend.Controller;
+
+import com.backend.Model.FavoriteBook;
+import com.backend.Service.FavoriteBookService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin("http://localhost:3000")
+@RestController
+@RequestMapping("/api/favorite_books")
+public class FavoriteBookController {
+
+    private final FavoriteBookService favoriteBookService;
+
+    public FavoriteBookController(FavoriteBookService favoriteBookService) {
+        this.favoriteBookService = favoriteBookService;
+    }
+
+    @PostMapping("/add")
```

Comment:
I was thinking about adding permissions for only user and admin roles to add, remove, and get favorite books. What do you think?

Reply:
I added those privilages